cpu/nrf52/periph/usbdev.c

```diff
@@ -275,7 +275,9 @@ static void _init(usbdev_t *dev)
     usbdev->sstate = NRFUSB_SETUP_READY;
 
     /* Enable a set of interrupts */
-    usbdev->device->INTEN = USBD_INTEN_USBRESET_Msk | USBD_INTEN_EPDATA_Msk;
+    usbdev->device->INTEN = USBD_INTEN_USBRESET_Msk |
```

Comment:
it's better if you put parenthesis around mask values:
```suggestion
    usbdev->device->INTEN = (USBD_INTEN_USBRESET_Msk |
                             USBD_INTEN_EPDATA_Msk |
                             USBD_INTEN_USBEVENT_Msk);
```

Reply:
Also fixed